management/src/clusterm/manager/client_test.go

```diff
@@ -30,6 +30,21 @@ var (
 	testReqBody   = APIRequest{
 		Nodes: []string{testNodeName},
 	}
+
+	testReqCommissionWithNodesBody = APIRequest{
+		Nodes:     []string{testNodeName},
+		HostGroup: ansibleMasterGroupName,
+	}
+
+	testReqCommissionWithoutNodesBody = APIRequest{
+		Nodes:     []string{},
+		HostGroup: ansibleMasterGroupName,
+	}
+
+	testReqCommissionWithoutHostGroupBody = APIRequest{
+		HostGroup: "",
```

Comment:
we should not need to set `host-group`. I think `omitempty` comment above will affect this as well.